Governance/tech/conventions.md

```diff
@@ -77,29 +82,29 @@ The time coordinate is defined by the `units` and the `calendar` attributes. The
 
 * A time coordinate variable should have a calendar attribute.
 * The use of a reference time in the year 0 to indicate climatological time is deprecated. 
-* Year and month should not be used as units, because of the potential for mistakes and confusion.
+* Year and month should not be used as `units`, because of the potential for mistakes and confusion.
 
 ```{warning}
 CF standards follows UDUNITS definition of a year to be exactly 365.242198781 days, and a month to be exactly year/12. These are different from a calendar year and a calendar month.
 ```
 
 ```{dropdown} **calendar** 
 **CF - recommended**<br>
-The [calendar](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#calendar) attribute defines the calendar used by the time coordinate. In order to calculate a time coordinate value from a date/time, or the reverse, one must know the units attribute of the time coordinate variable (containing the time unit of the coordinate values and the reference date/time) and the calendar.<br>
+The [calendar](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#calendar) attribute defines the calendar used by the time coordinate. In order to calculate a time coordinate value from a date/time, or the reverse, one must know the `units` attribute of the time coordinate variable (containing the time unit of the coordinate values and the reference date/time) and the calendar.<br>
```

Comment:
I read this in the previous commit, too; not sure if double?

Reply:
I can't find it twice, but as this is still a draft with many holes we'll have a chance to check again